### ISSUE TITLE
SCM viewlet click/keyboard navigation doesn't work

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#21200]
#20999

Clicking always forces focus in the right-side editor so it's only possible to navigate the viewlet list if you navigate focus there (eg. click commit message and tab to the list).

**Comment by joaomoreno** [comment]
#20909
